core/lib/zksync_core/src/api_server/web3/state.rs

```diff
@@ -578,13 +625,24 @@ impl Filters {
 
     /// Retrieves filter from the state.
     pub fn get(&self, index: U256) -> Option<&TypedFilter> {
-        self.state.get(&index)
+        let installed_filter = self.state.get(&index)?;
+
+        Some(&installed_filter.filter)
+    }
+
+    pub fn update_stats(&mut self, index: U256) -> bool {
+        if let Some(installed_filter) = self.state.get_mut(&index) {
+            installed_filter.update_stats();
+            true
```

Comment:
Nit: You don't seem to use the returned value anywhere, so it can be removed. Ditto for `update()`.